AK/Bitmap.h

```diff
@@ -156,7 +156,7 @@ class Bitmap {
             if (!get(j))
                 return j;
 
-        return -1;
+        return {};
```

Comment:
@awesomekling I'm still not very sure if this change is necessary.

Reply:
This change is great, it's fixing a bug!
However, it's the default constructor of `Optional`, not `Bitmap`. :)

Reply:
Oops, my bad, I'll fix it :)